open-sphere-plugins/csv/src/main/java/io/opensphere/csv/CSVTypeInfoGenerator.java

```diff
@@ -77,7 +78,8 @@ public static CSVDataTypeInfo generateTypeInfo(Toolbox tb, CSVDataSource fileSou
         typeInfo.applyColorPreferences();
 
         // Set map visualization info
-        if (fileSource.getParseParameters().hasCategory(ColumnType.Category.SPATIAL))
+        if (fileSource.getParseParameters().hasCategory(ColumnType.Category.SPATIAL)
+                || fileSource.getParseParameters() instanceof CSVParseParameters)

```

Comment:
What's the point of checking instanceof CSVParseParameters?  I thought that would always be true.

Reply:
The idea was to let CSVs that don't have a SPATIAL column loaded still have MapVisualization loaded. The would let, for example, a CSV with only column headings and no data be loaded in as a feature and not an unknown. I wasn't sure if the instanceof check would always be true. I could look into seeing if there are other parameters that could fit instead, or just do away with the conditional in the extreme case if that wouldn't break something.